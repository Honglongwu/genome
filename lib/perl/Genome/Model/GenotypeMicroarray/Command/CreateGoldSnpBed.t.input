../../../../../../test-data-relocate-me/lib--perl--Genome--Model--GenotypeMicroarray--Command--CreateGoldSnpBed.t.input