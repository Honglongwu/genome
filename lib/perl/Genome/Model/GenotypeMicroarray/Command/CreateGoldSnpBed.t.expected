../../../../../../test-data-relocate-me/lib--perl--Genome--Model--GenotypeMicroarray--Command--CreateGoldSnpBed.t.expected